src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4808,6 +4808,105 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
     Add-Content -Path $pluginConfigFile -Value ""CORECLRDIR=$PSHOME""
 }}
 
+function Copy-PluginToEndpoint
+{{
+param(
+    [Parameter()] [string] $endpointDir
+)
+    $resolvedPluginInstallPath = """"
+    $pluginInstallPath = Join-Path ""$env:WINDIR\System32\PowerShell"" $endpointDir
```

Comment:
Use `[System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::Windows)` instead of $env:windir.  This won't work on nanoserver, but remoting doesn't work on nanoserver.

Reply:
Done.